rfcs/web_features_known_issues.md

```diff
@@ -0,0 +1,58 @@
+## Summary
+
+This RFC proposes the creation of a new repository within the web-platform-tests organization to manage information about known issues affecting web platform tests. The repository will provide a transparent and collaborative space for documenting these issues. The data in this repository aims to be machine readable source of truth for sites or tools that make use of the web-features grouping of WPT such as webstatus.dev.
+
+## Details
+
+* **Repository Name:** web-platform-tests-feature-issues
```

Comment:
Any repo name will be OK, but in terms of tab completion, I wonder if web-features-issues would be acceptable?